packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -120,6 +129,7 @@ function GridItem< Item >( {
 		<VStack
 			spacing={ 0 }
 			key={ id }
+			role="gridcell"
```

Comment:
It won't make a difference in terms of what gets rendered in the browser, but I think that adding `role="gridcell"` to `Composite.Item` would be more clear, given that all other roles are also applied to `Composite` components?